<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KSJDemoQT_Base</class>
 <widget class="QDialog" name="KSJDemoQT_Base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KSJCalibration</string>
  </property>
  <widget class="QWidget" name="PREVIEWWND_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>501</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="DEVICENUM_Label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Device Num:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DEVICE_LIST_ComboBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="ExposureLinesSpinBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>100</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="GainSpinBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>170</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Gain:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure Time(ms):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure Lines:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ExposureTimeSpinBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range(ms):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>170</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range:</string>
   </property>
  </widget>
  <widget class="QLabel" name="EXPOSURE_TIME_RANGE_Label">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range</string>
   </property>
  </widget>
  <widget class="QLabel" name="EXPOSURE_LINES_RANGE_Label">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range</string>
   </property>
  </widget>
  <widget class="QLabel" name="GAIN_RANGE_Label">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range</string>
   </property>
  </widget>
  <widget class="QLabel" name="EXPOSURE_TIME_REAL_Label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>140</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>REAL EXP TIME</string>
   </property>
  </widget>
  <widget class="QLabel" name="FOV_RANGE_Label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>200</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>EXP LINES RANGE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>200</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>FOV Range:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Capture</string>
   </property>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Col Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Row Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Col Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Row Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MultiFrame:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetCaptureFovButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Capture Fov</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="CaptureColStartSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="CaptureColSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="CaptureRowStartSpinBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="CaptureRowSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="CaptureMultiFrameSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="PreviewButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CaptureButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QLabel" name="ERROR_INFO_Label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>380</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Error Info</string>
   </property>
  </widget>
  <widget class="QLabel" name="ELAPSE_TIME_Label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>380</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Elapse</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="FunctionTableWidget">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>0</y>
     <width>311</width>
     <height>641</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="SaveCheckBox">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>310</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateMapButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>610</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Map</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CaliCheckBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>610</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibration</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ChessWidthSpinBox">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>610</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>4</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="ChessHeightSpinBox">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>610</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="ERROR_INFO_Label_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>610</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Chess Size:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
